examples/src/examples/graphics/index.mjs

```diff
@@ -8,7 +8,7 @@ import ClusteredLightingExample from "./clustered-lighting";
 import ClusteredOmniShadowsExample from "./clustered-omni-shadows";
 import ClusteredSpotShadowsExample from "./clustered-spot-shadows";
 import ContactHardeningShadowsExample from "./contact-hardening-shadows";
-import CustomMaterialExample from "./custom-material";
+import LitMaterialExample from "./lit-material";
```

Comment:
we probably want to remove this to avoid the example going public till it's all ready